lib/Conversion/TorchToLinalg/DataMovement.cpp

```diff
@@ -317,15 +317,15 @@ class ConvertAtenViewOp : public OpConversionPattern<AtenViewOp> {
       if (inputDynamicValues) {
         int64_t missingValue = outputProduct / inputProduct;
         for (size_t i = 0; i < inputShape.size(); i++) {
-          if (inputShape[i] == -1) {
+          if (inputShape[i] == kUnknownSize) {
             inputShape[i] = missingValue;
             break;
           }
         }
       } else {
         int64_t missingValue = inputProduct / outputProduct;
         for (size_t i = 0; i < outputShape.size(); i++) {
-          if (outputShape[i] == -1) {
+          if (outputShape[i] == kUnknownSize) {
```

Comment:
This is just plain wrong. The PyTorch API specifies that the value is -1 here. It needs to be -1 and not the internal constant.